### PR TITLE
Support arbitrary parameters in spii

### PR DESCRIPTION
**Pull request by alextargov**
**Description**

Changes proposed in this pull request:
- Introduce a new field in the Webhook entity - Parameters
- Add migrations
- Adapt GQL and GQLizers 

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

**Comment by alextargov** [comment]
reset-vm

File: components/director/pkg/graphql/graphqlizer/graphqlizer.go [hunk 1]
```diff
@@ -365,6 +365,9 @@ func (g *Graphqlizer) WebhookInputToGQL(in *graphql.WebhookInput) (string, error
 		{{- if .StatusTemplate }}
 		statusTemplate: "{{.StatusTemplate }}",
 		{{- end }}
```
Comment (kaloyanspiridonov) [inline components/director/pkg/graphql/graphqlizer/graphqlizer.go:367]: update director dependency in all projects

**Review by kaloyanspiridonov — CHANGES_REQUESTED** [review]

**Review by kaloyanspiridonov — APPROVED** [review]

**Review by la4ezar — APPROVED** [review]

**Pull request merged**